### ISSUE TITLE
Feature - See the content of stored mapping files

### ISSUE THREAD

**Issue by ihcomega56** [pyrsia/pyrsia#1593]
### Suggestion details

Related to #1350

Do we need a feature which enable us to see or download mapping files published in our P2P network? It may be needed for troubleshooting or investigating something.
This issue is created not to forget to think about it.

**Comment by betarelease** [comment]
Since this ends up on the blockchain it will be part of the transparency log. We need an and cli command to allow access to it.

**Comment by ihcomega56** [comment]
Yes, so this issue is for the feature which allows us to see the mapping file itself (Probably transparency log wouldn't give us everything in the file).
